cynthion/python/src/__init__.py

```diff
@@ -20,6 +25,9 @@
 from . import gateware
 from . import shared
 
+import importlib.metadata
+__version__ = importlib.metadata.version(__package__)
```

Comment:
`importlib` always scares me in the particular range of python versions we operate in, has this been tested on a clean Python 3.8 install?

I vaguely remember having to do a `from importlib.metadata import version` or add a: `"importlib-metadata; python_version<'3.9'",` to the project dependencies but that could have been on Python 3.7…

Reply:
Yes, `importlib.metadata` has been included in stdlib since 3.8. I've tested with 3.8.

Reply:
You're probably remembering what you did with `importlib.resources`, not `importlib.metadata`.